examples/objectdetection/src/CameraStream.tsx

```diff
@@ -0,0 +1,208 @@
+import {
+  Canvas,
+  Group,
+  Rect,
+  Text as SkiaText,
+  matchFont,
+} from "@shopify/react-native-skia";
+import * as React from "react";
+
+import {
+  Platform,
+  Pressable,
+  StyleSheet,
+  Text,
+  View,
+  useWindowDimensions,
+} from "react-native";
+import {
+  Delegate,
+  MediapipeCamera,
+  RunningMode,
+  useObjectDetection,
+} from "react-native-mediapipe";
+
+import {
+  useCameraPermission,
+  useMicrophonePermission,
+} from "react-native-vision-camera";
+
+interface Detection {
+  label: string;
+  x: number;
+  y: number;
+  width: number;
+  height: number;
+}
+
+export default function CameraStream(): React.ReactElement | null {
```

Comment:
I am not really a fan of default exports. And specifying the function in this way is, indeed, not 'react-y'

```
type Props = BottomTabScreenProps<RootTabParamList,"CameraStream">;
export const CameraStream: React.FC<Props> = () => { ...
```
What you are doing works fine now, but would work less well under different circumstances, like if you wanted to navigate inside this component, or if the component took parameters.

Reply:
Thanks.